### DISCUSSION TITLE
Google hack account

### DISCUSSION THREAD

**Discussion by Heliodoro052482** [community/community#113428] [Other Feature Feedback, Questions, & Ideas]
I'm needing secure advance my Google accounts are being breached on devices I'm not using I only have Nokia phone and there is a duplicate need location real time to secure

**Comment by SebastienXia** [comment]
Hello, @Heliodoro052482 

Github Community isn't the Google support so please, report your problem in this link : https://support.google.com/?hl=en

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Heliodoro052482! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, @SebastienXia's response above is correct, you will need to reach out to Google in order to get support with your Google account. 

Good luck!
